src/components/Button/Button.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { HTMLProps } from "react";
+
+import "./_button.scss";
+
+type Props = {
+  onClick: () => void;
+  children: string;
+} & HTMLProps<HTMLElement>;
```

Comment:
Because we aren't adding any html props to these components we don't need this `& HTMLProps<HTMLElement>` definition here, or in the `Banner` component, can you remove it from both?

```suggestion
};
```